src/components/QueryBuilder.vue

```diff
@@ -10,26 +10,9 @@
 		<div role="form">
 			<h2 class="querybuilder__find-title"
 				v-i18n="{msg: 'query-builder-find-all-items'}" />
-			<div class="querybuilder__rule">
-				<PropertyLookup
-					v-model="selectedProperty"
-					:error="fieldErrors.property"
-				/>
-				<ValueTypeDropDown
-					v-model="selectedPropertyValueRelation"
-					:disabled="limitedSupport"
-				/>
-				<TextInput
-					class="querybuilder__rule__value"
-					:label="$i18n('query-builder-input-value-label')"
-					ref="value"
-					v-model="textInputValue"
-					:error="fieldErrors.value ?
-						{message: $i18n(fieldErrors.value.message), type: fieldErrors.value.type}: null"
-					:placeholder="$i18n('query-builder-input-value-placeholder')"
-					:disabled="selectedPropertyValueRelation === propertyValueRelation.Regardless"
-				/>
-			</div>
+			<QueryCondition
+				ref="firstCondition"
```

Comment:
Are we using this somewhere?

Reply:
No, I put it for later but we can add it later too